### ISSUE TITLE
Old 3DS - Online Updater not working.

### ISSUE THREAD

**Issue by Dougzen** [libretro/RetroArch#12658]
Everytime that I tried use this function, the app freeze.
I wanted this to add thumbnails.

**Comment by Syko7956** [comment]
Yeah I don't know if it's current builds but if I grab any cores from the online updater, it's just gonna freeze. I have to stay on 1.9.6 Stable cores

**Comment by Syko7956** [comment]
If you download a core and boot it right away, it will black screen. But will work after restart. Before it crashed. I recall it didn't need a system restart before tho.

**Comment by LibretroAdmin** [comment]
@MrHuu Is this still an issue on current nightlies?

**Comment by MrHuu** [comment]
This is still an ongoing issue.

I'm not really sure if the o3DS has it's own specific issues.
But currently, when having the threaded video driver enabled, most ```Online Updater``` tasks cause the 3DS to freeze.

When having the threaded video driver disabled, the 3DS does not freeze in my experience.
I believe this is a threading related issue.

Unfortunately, having the threaded video driver enabled is required in most cases to obtain full speed thus having it enabled would be the default for most users.

**Comment by dmaglio** [comment]
The bug is still present. With any voice from the _Online updater_ section, the 3DS freezes. I've also tried enabling the log, but it doesn't save anything to the file.

**Comment by pstef** [comment]
Should be fixed by https://github.com/libretro/RetroArch/pull/17735.
